releasenotes/notes/28742.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: release-notes/v2
+kind: feature
+
+area: security
+
+issue:
+    - https://github.com/istio/istio/issues/28742
+
+releaseNotes:
+- |
+  **Added** Configuring Envoy to fetch the Jwks if the JwksUri is a mesh cluster URL for mTLS and other benefits. This is disabled by default 
```

Comment:
This isn't very clear, reading this I don't know why I should care. When would a user want to enable this?

Reply:
>  When would a user want to enable this?
"if the JwksUri is a mesh cluster URL for mTLS and other benefits" This is the answer. Currently if Jwks server is a mesh cluster it is not supported. Let me know if I can make it more clear. I will change.

Reply:
reworded a bit. See if it helps